package.json

```diff
@@ -56,7 +56,7 @@
   "scripts": {
     "prebuild": "npm run clean",
     "build": "hardhat compile --force",
-    "postbuild": "tsc --project tsconfig.build.json",
+    "postbuild": "tsc --project tsconfig.build.json && cp -R contracts dist/",
```

Comment:
shouldn't be better to remove it from the `.npmignore`, so we can include it, but not in the dist folder?